tutorials/customizing-base-map-styles/index.md

```diff
@@ -0,0 +1,379 @@
+---
+title: Customizing base map styles
+description: If you are getting bored of looking at the same two default basemaps in Earth Engine, or need to use a specific visualization for functionality/convenience, this tutorial will show you how to edit all of its characteristics.
```

Comment:
"Base map" seems to be the preferred term - please fix "basemap" here and below, for consistency.

Reply:
Done